### ISSUE TITLE
Create `--quiet` flag for CLI 

### ISSUE THREAD

**Issue by Veinar** [Veinar/envcloak#10]
Add `--quiet` flag to supress any `print()`.
This is total opposite of #8.

Steps TODO:

* Add flag in cli.py as it was done in for example --no-gitignore
* Pass this flag to encryptor.py and/or validation.py
* Omit all print statements describing operations as long as the flag is True.

🚫 **Warning:** This issue should not be implemented in parallel with #8, it may cause a conflict of interest

**Comment by Veinar** [comment]
Making it `ON HOLD` due to very low output of commands, just one line per command it should have less priority than #8.

**Comment by Veinar** [comment]
Issue is back on menu. Due to completion of #8.

**Comment by Failte327** [comment]
I can work on this later today if you want to assign it to me 🙂

**Comment by Veinar** [comment]
Hi @Failte327,

Thank you for your willingness to help, I assign the issue to you.

**Comment by Failte327** [comment]
Sorry, got caught up with other things the last few days. I did some initial work on this but haven't completed a good solution yet. Should be able to in the next day or two

**Comment by Veinar** [comment]
Sure no problem, thanks for letting me know  ;)

**Comment by Veinar** [comment]
Hi @Failte327 any update ?
